src/swig/CMakeLists.txt

```diff
@@ -28,3 +34,17 @@ SWIG_LINK_LIBRARIES(${PROJECT_NAME}
     fastrtps
     )
 
+# Find the installation path
+execute_process(COMMAND ${PYTHON_EXECUTABLE} -c "from distutils import sysconfig; print(sysconfig.get_python_lib(plat_specific=True, prefix='${CMAKE_INSTALL_PREFIX}' ) )"
+    OUTPUT_VARIABLE _ABS_PYTHON_MODULE_PATH
+    OUTPUT_STRIP_TRAILING_WHITESPACE
+    )
+get_filename_component (_ABS_PYTHON_MODULE_PATH ${_ABS_PYTHON_MODULE_PATH} ABSOLUTE)
+file (RELATIVE_PATH _REL_PYTHON_MODULE_PATH ${CMAKE_INSTALL_PREFIX} ${_ABS_PYTHON_MODULE_PATH})
+SET (PYTHON_MODULE_PATH
+    ${_REL_PYTHON_MODULE_PATH}
+    )
+
+# Install
+INSTALL ( FILES ${CMAKE_CURRENT_BINARY_DIR}/_${PROJECT_NAME}.so DESTINATION ${PYTHON_MODULE_PATH} )
+INSTALL ( FILES ${CMAKE_CURRENT_BINARY_DIR}/${${PROJECT_NAME}_MODULE}.py DESTINATION ${PYTHON_MODULE_PATH} )
```

Comment:
```suggestion
INSTALL ( FILES $<TARGET_FILE:${PROJECT_NAME}> DESTINATION ${PYTHON_MODULE_PATH} )
INSTALL ( FILES $<TARGET_FILE_DIR:${PROJECT_NAME}>/${${PROJECT_NAME}_MODULE}.py DESTINATION ${PYTHON_MODULE_PATH} )
```

Reply:
@richiware Does doing this mean that the module would need to be imported doing `import fastdds_module`? Furthermore, would [this](https://github.com/eProsima/Fast-DDS-python/blob/bugfix/fixes/src/swig/fastdds.i#L1) need to be updated? Just asking cause I'd really like the import to be just `fastdds`

Reply:
Never mind

Reply:
Done